addons/website_slides/static/src/xml/activity.xml

```diff
@@ -19,4 +19,32 @@
             </t>
         </t>
     </t>
+
+    <t t-name="slides.Slide" owl="1">
+        <div t-if="slide" class="o_Slide">
```

Comment:
split `t-if` into separate ` <t/>` tag

Also OWL components must have exactly one single DOM element as root, and it cannot be conditional.